lib/gemstash/configuration.rb

```diff
@@ -46,5 +47,19 @@ def default?(key)
     def [](key)
       @config[key]
     end
+
+  private
+
+    def default_file
+      File.exist?("#{DEFAULT_FILE}.erb") ? "#{DEFAULT_FILE}.erb" : DEFAULT_FILE
+    end
+
+    def parse_config(file)
+      if File.fnmatch?("**.erb", file)
+        @config = YAML.load(::ERB.new(File.read(file)).result)
```

Comment:
Would you mind dropping the `::` from `::ERB`? Unless I'm missing something, I believe it won't be necessary.